src/vs/workbench/contrib/searchEditor/browser/searchEditorActions.ts

```diff
@@ -173,12 +173,14 @@ export const openNewSearchEditor =
 			editor = await editorService.openEditor(input, { pinned: true }, toSide ? SIDE_GROUP : ACTIVE_GROUP) as SearchEditor;
 		}
 
+		editor.setFocusResults(args.focusResults);
```

Comment:
I'm not sure how this part helps to fix the bug, since the `args.focusResults` was being passed in previously anyways. Why does this need to be made a class variable?

Reply:
The only change that I see making a difference was the one found here: https://github.com/microsoft/vscode/pull/205914/files#diff-9cc7738b5c4fb266d70547a9cc8b7f6ea60116870c16a677f6658059c768aa45R484-R485, but please correct me if I'm wrong.

Reply:
> I'm not sure how this part helps to fix the bug, since the `args.focusResults` was being passed in previously anyways. Why does this need to be made a class variable?

You're right, we might just use casting like this:
```C
const focusResults = (this.priorConfig as LegacySearchEditorArgs).focusResults;
```

> The only change that I see making a difference was the one found here: https://github.com/microsoft/vscode/pull/205914/files#diff-9cc7738b5c4fb266d70547a9cc8b7f6ea60116870c16a677f6658059c768aa45R484-R485

Yes, this is the most important change. All other changes are needed to make the `focusResults` option to work in the use case that was hardcored to `true` here:
https://github.com/microsoft/vscode/blob/313c78eeadbce621c3a5e890b4d1760bb21d7a46/src/vs/workbench/contrib/searchEditor/browser/searchEditorActions.ts#L222